PyPDF2/pdf.py

```diff
@@ -498,8 +498,11 @@ def write(self, stream):
                 assert len(key) == (len(self._encrypt_key) + 5)
                 md5_hash = md5(key).digest()
                 key = md5_hash[:min(16, len(self._encrypt_key) + 5)]
-            obj.writeToStream(stream, key)
-            stream.write(b_("\nendobj\n"))
+            try:
+                obj.writeToStream(stream, key)
+                stream.write(b_("\nendobj\n"))
+            except:
+                pass
```

Comment:
```suggestion
            except Exception as e:
                if self.strict:
                    raise e
```

Reply:
At the very least I want the strict mode to give an exception ... and the non-strict probably a logging message.